test/extended/olm/olmv1.go

```diff
@@ -72,7 +73,7 @@ var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM] OLMv1 CRDs", func() {
 	})
 })
 
-var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 Catalogs", func() {
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 default Catalogs", func() {
```

Comment:
Do we know what the impact of changing the test name is?

Reply:
Probably disconnected past history of this test from new history after this merges. I'm not sure adding the word `default` is justification enough for the potential implications.